src/ext/java/org/opentripplanner/ext/actuator/ActuatorAPI.java

```diff
@@ -128,13 +127,13 @@ public Response actuator(@Context UriInfo uriInfo) {
     @Path("/health")
     public Response health() {
         if (router.graph.updaterManager != null) {
-            Collection<String> waitingUpdaters = router.graph.updaterManager.waitingUpdaters();
+            var waitingForUpdaters = router.graph.updaterManager.listNonePrimedUpdaters();
```

Comment:
I think you should change the method name to `listUnprimedUpdaters`. I was a little confused at first.